README.md

```diff
@@ -29,6 +29,7 @@ The table below shows the exact version of ACS deployed with each chart version/
 |--|--|--|--|--|--|
 | 5.0.0-M1 | 7.0.0-M2 | 6.2.2 | 6.1.1 | 6.0.1 | 6.2.1-A8 |
 | 5.0.0-M2 | 7.0.0-M3 | 6.2.2 | 6.1.1 | 6.0.1 | 7.0.0-A10 |
+| 5.0.0-M3 | 7.0.0-A20 | 6.2.2 | 6.1.1 | 6.0.1 | 7.0.0-A20 |
```

Comment:
We normally add this at the time of the release so it matches to the available tags.

Reply:
Removed line